.github/workflows/main.yml

```diff
@@ -13,17 +11,50 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node: [ '10', '12', '14', '16' ]
 
     steps:
       - uses: actions/checkout@v2
 
       - name: Setup node
         uses: actions/setup-node@v2
+
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "::set-output name=dir::$(yarn cache dir)"
```

Comment:
Should we be sharing caches between test and build configurations?

Reply:
I don't feel super strongly about it. I wanted to use same packages in both. Not sure how much does it matter with .lock file.

Reply:
It doesn't cache node_modules, just the global cache so we save on not downloading packages from npm.